<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitcoinTests</class>
 <widget class="QMainWindow" name="BitcoinTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bitcoin Tests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_Main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>931</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_BitcoinAddressTests">
     <attribute name="title">
      <string>Bitcoin Address Tests</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_PrivateKeyTests">
     <attribute name="title">
      <string>Private Key Tests</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_PrivateKeyTests">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_WIFToPrivKey">
       <attribute name="title">
        <string>Wallet import format to private key</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_PrivKeyToWIF">
       <attribute name="title">
        <string>Private key to wallet import format</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_PrependVersion">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_First4BitesOf4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>5. First 4 bytes of 4:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_CalculateWIF">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_PraseToECDSAQt">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>10</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Prase =&gt; Priv ECDSA (Bitcoin)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_AfterBase58">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_SHA256HashOf2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>3. SHA-256 hash of 2:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_PrivECDSAKey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1. Private ECDSA Key:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SHA256HashOf3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Phrase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>37</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Phrase:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_SHA256HashOf2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>6. Adding 5 at the end of 2:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PrivECDSAKey">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>711</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_PrependVersion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>2. Prepend version (add 0x80):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Phrase">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>541</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_First4BitesOf4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>190</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_SHA256HashOf3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>4. SHA-256 hash of 3:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>7. Base58 encoding of 6:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_WIFChecksumChecking">
       <attribute name="title">
        <string>WIF checksum checking</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_VanityTests">
     <attribute name="title">
      <string>Vanity Tests</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_Brainwallet">
     <attribute name="title">
      <string>Brainwallet</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_ProfOfBurn">
     <attribute name="title">
      <string>Proof of Burn</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_SHA256">
     <attribute name="title">
      <string>Libs Tests</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_LibsTests">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_HashCalculator">
       <attribute name="title">
        <string>Hash Encode Calculator</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_CalcStringHash">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>250</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate String Hash</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_DataForHash">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>881</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_HashResult">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>881</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_DataForHashToLower">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Lower</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_DataForHashToUpper">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Upper</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_DataForHash">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_HashResult">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Hash:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_HashType">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_HashFunction">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_HashToLower">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Lower</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_HashToUpper">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Upper</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_CalcHexHash">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>250</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate Hex Hash</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_PublicECDSAKey">
       <attribute name="title">
        <string>Public ECDSA Key</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_PrivECDSAKeyTests">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>771</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_PrivECDSAKeyTests">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string> Private ECDSA Key:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_CalcPublicECDSAKeyTests">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>40</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_PublicECDSAKeyTestsUncompressed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Public ECDSA Key (uncompressed):</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PublicECDSAKeyTestsUncompressed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>881</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_PublicECDSAKeyTestsCompressed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>881</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_PublicECDSAKeyTestsCompressed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Public ECDSA Key (compressed):</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_PublicKeyECDAKeyToUpper">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Upper</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_PublicKeyECDAKeyToLower">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>to Lower</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
